lib/carrierwave-aws.rb

```diff
@@ -43,7 +44,33 @@ def self.normalized_acl(acl)
     normalized
   end
 
+  def self.aws_signer(value = nil)
+    self.aws_signer = value if value
```

Comment:
Why is the value being set during the caller? The side effect of storing a value seems unexpected to me.